automated-comments/javascript/resistor-color-duo/must_use_a_helper.md

```diff
@@ -0,0 +1,2 @@
+Using a helper method is good practice, because it replaces a cryptic "member
+call" with a named call, that can be documented individually.
```

Comment:
```suggestion
call" with a named call that can be documented individually.
```

In English, if there's a comma, use "which". If there's not a comma, use "that". Use "which" when the sentence would still make sense without the clause that follows it, and use that if it wouldn't.

So here, as "can be documented individually" is part of the reason, and things don't make sense without it, I'd use "that" and therefore no comma.